samples/nrf9160/http_application_update/prj.conf

```diff
@@ -0,0 +1,61 @@
+# General config
+CONFIG_NEWLIB_LIBC=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_ASSERT=y
+CONFIG_REBOOT=y
+
+# Trusted execution
+CONFIG_TRUSTED_EXECUTION_NONSECURE=y
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# LTE link control
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_AUTO_INIT_AND_CONNECT=y
+
+# BSD library
+CONFIG_BSD_LIBRARY=y
+
+# Library for buttons and LEDs
+CONFIG_DK_LIBRARY=y
+CONFIG_DK_LIBRARY_INVERT_LEDS=n
+
+# Console
+CONFIG_CONSOLE_SUBSYS=y
+CONFIG_CONSOLE_HANDLER=y
+CONFIG_CONSOLE_GETCHAR=y
+
+# Main thread
+CONFIG_MAIN_THREAD_PRIORITY=7
+
+# Heap and stacks
+CONFIG_HEAP_MEM_POOL_SIZE=16384
+CONFIG_MAIN_STACK_SIZE=8192
+CONFIG_MPU_STACK_GUARD=y
+
+CONFIG_DEBUG=y
```

Comment:
This will `Build kernel with debugging enabled`. Is this intended?

Reply:
Removed